assets/app.js

```diff
@@ -68,6 +68,44 @@ var settingStorage = {
     }
 }
 
+Vue.component("metatags-modal", {
+   template: '#metatags-modal-template',
+   props: ['cell', 'field', 'close', 'save'],
+   data: function () {
+       return {
+           items: [],
+           newItem: ''
+       }
+   },
+    mounted: function () {
+       this.$nextTick(function () {
+           if (this.field && this.cell.row[this.field]) {
+                this.items = this.cell.row[this.field].split(',');
+           }
+       })
+    },
+    methods: {
+       add: function () {
+           if (this.newItem) {
+               this.items.push(this.newItem);
+               this.newItem = '';
+           }
+       },
+        remove: function (index) {
+           this.items.splice(index, 1);
+        },
+        saveToDB: function () {
+            const result = this.items.filter(function (el) {
+                return el != null;
+            });
+            this.cell.row[this.field] = result.join(',');
+
+            this.save('metatags', this.cell);
+            this.close();
+        }
+
+    }
```

Comment:
indentation
![image](https://user-images.githubusercontent.com/13611391/97938048-f3681180-1db2-11eb-9d54-ed06a7f2bf81.png)

Reply:
data: function() && mounted: function() are the same layers with the methods: {} 

Edit: I will change the sequence to make it clearer.
![image](https://user-images.githubusercontent.com/10362733/97948776-22d34a00-1dc4-11eb-8442-ede0e1a1a86c.png)

Reply:
...

Reply:
![image](https://user-images.githubusercontent.com/13611391/97966255-2da2d480-1dee-11eb-9b31-8b3a0b80c6f3.png)

add มันมี 3 spaces ก่อนหน้า มึงเห็นมั้ย